src/internal/ppsutil/decoder.go

```diff
@@ -1,18 +1,22 @@
 package ppsutil
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
 	"os"
 	"strings"
 
+	"google.golang.org/protobuf/encoding/protojson"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/reflect/protoreflect"
 	"gopkg.in/yaml.v3"
 
+	"github.com/pachyderm/pachyderm/v2/src/pps"
+	ppsclient "github.com/pachyderm/pachyderm/v2/src/pps"
```

Comment:
You are already importing this as not ppsclient.

Reply:
Thanks, I hadn’t noticed that. I replaced `ppsclient` with `pps` throughout.